Formula/a/astro.rb

```diff
@@ -20,6 +20,9 @@ class Astro < Formula
   end
 
   depends_on "go" => :build
+  on_macos do
+    depends_on "podman"
+  end
```

Comment:
it is only for macos?

Reply:
Yes, podman dependency is only limited to macos